python/ray/autoscaler/azure/defaults.yaml

```diff
@@ -130,7 +130,7 @@ setup_commands:
     - echo 'eval "$(conda shell.bash hook)"' >> ~/.bashrc
     # - echo 'conda activate py37_pytorch' >> ~/.bashrc
     - echo 'conda activate py37_tensorflow' >> ~/.bashrc
```

Comment:
I was also wondering if this is a valid conda environment?

Reply:
yes, both of those envs are valid in the dsvm